formats/cbor/commonMain/src/kotlinx/serialization/cbor/Cbor.kt

```diff
@@ -34,12 +34,12 @@ public sealed class Cbor(
             CborConfiguration(
                 encodeDefaults = false,
                 ignoreUnknownKeys = false,
-                encodeKeyTags = false,
-                encodeValueTags = false,
-                encodeObjectTags = false,
-                verifyKeyTags = false,
-                verifyValueTags = false,
-                verifyObjectTags = false,
+                encodeKeyTags = true,
+                encodeValueTags = true,
+                encodeObjectTags = true,
+                verifyKeyTags = true,
+                verifyValueTags = true,
+                verifyObjectTags = true,
```

Comment:
As these parameters change the behaviour of the system you should not change the default. Instead you could provide a way for users to choose a newer/different configuration.

Reply:
thanks! definitely not intended